addon/PreviewProvider.js

```diff
@@ -46,6 +46,13 @@ function PreviewProvider(tabTracker, metadataStore, store, options = {}) {
   this.init();
 }
 
+function createCacheKey(spec) {
```

Comment:
Since we removed create-cache-key.js, maybe revert this change to pulling out createCacheKey?